{{cookiecutter.package_slug}}/pyproject.toml

```
@@ -1,9 +1,12 @@
 [project]
 name = "{{ cookiecutter.package_name }}"
+{% if cookiecutter.authors != "" %}
 authors = [
-    {name = "Adam Pirog"},
-    {email = "pirog.adam@gmail.com"},
+    {% for author in cookiecutter.authors.split(',') %}
```

```suggestion
    {% for author in cookiecutter.authors.split(',') -%}
```
Authors get too spread out.

Really nice touch with the loop tho